tools/InstallToVSCode/InstallToVSCode.cmd

```diff
@@ -92,22 +92,28 @@ for %%f in (xunit.console.netcore.exe) do call :InstallFile "%OpenDebugAD7BinDir
 for %%f in (%OpenDebugAD7BinDir%\*.dll) do call :InstallFile "%%f"
 
 echo.
-echo Installing clrdbg bits from %CLRDBGBITSDIR%...
+echo Installing vsdbg bits from %VSDBGBITSDIR%...
 
 REM NOTE: We ignore files that already exist. This is because we have already
 REM cleaned the directory originally, and published CoreCLR files. Replacing existing
 REM files will replace some of those CoreCLR files with new copies that will not work.
-for %%f in (%CLRDBGBITSDIR%\*.dll) do call :InstallNewFile "%%f"
-for %%f in (%CLRDBGBITSDIR%\*.exe) do call :InstallFile "%%f"
-for %%f in (%CLRDBGBITSDIR%\*.vsdconfig) do call :InstallFile "%%f"
-for %%f in (%CLRDBGBITSDIR%\version.txt) do call :InstallFile "%%f"
-for /D %%d in (%CLRDBGBITSDIR%\*) do (
+for %%f in (%VSDBGBITSDIR%\*.dll) do call :InstallNewFile "%%f"
+for %%f in (%VSDBGBITSDIR%\*.exe) do call :InstallFile "%%f"
+for %%f in (%VSDBGBITSDIR%\*.vsdconfig) do call :InstallFile "%%f"
+for %%f in (%VSDBGBITSDIR%\version.txt) do call :InstallFile "%%f"
+for /D %%d in (%VSDBGBITSDIR%\*) do (
     echo.
-    echo Installing clrdbg bits from %%d... to %%~nd
+    echo Installing vsdbg bits from %%d... to %%~nd
     if NOT exist "%DESTDIR%\%%~nd" mkdir "%DESTDIR%\%%~nd
     for %%f in (%%d\*.dll) do call :InstallFile "%%f" %%~nd\
 )
 
+REM Rename vsdbg back to clrdbg
```

Comment:
is it worth having this part behind an option so we can toggle whether to rename it or not?